contrib/pkg/deprovision/gcp.go

```diff
@@ -45,6 +47,7 @@ func NewDeprovisionGCPCommand() *cobra.Command {
 	flags := cmd.Flags()
 	flags.StringVar(&opt.logLevel, "loglevel", "info", "log level, one of: debug, info, warn, error, fatal, panic")
 	flags.StringVar(&opt.region, "region", "", "GCP region where the cluster is installed")
+	flags.StringVar(&opt.credsFile, "creds-file", "", "File containing GCP credentials")
```

Comment:
If we are going to add this for GCP deprovision (which I am not convinced that we should), then we should add it to all of the platforms.